### PR TITLE
fix(tabs-extended): item token update

### PR DESCRIPTION
**Pull request by NSDrowned**
### Related Ticket(s)

https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/6276

### Description

Added `body-long-01` to tab items. Selected items already using `productive-heading-01` and at 14px.

### CODE REVIEW

**Comment by NSDrowned** [comment]
@oliviaflory I fixed the text alignment I found 👍

File: packages/styles/scss/components/tabs-extended/_tabs-extended.scss [hunk 1]
```diff
@@ -48,6 +52,7 @@
             @include type-style('productive-heading-01');
 
             color: $text-primary;
+            line-height: 1.375rem;
```
Comment (annawen1) [inline packages/styles/scss/components/tabs-extended/_tabs-extended.scss:55]: The font tokens should already set the `line-height`, this is something we want to overwrite?

**Review by annawen1 — COMMENTED** [review]
code-wise looks good, just one question about the `line-height` property being set

File: packages/styles/scss/components/tabs-extended/_tabs-extended.scss (hunk 1 above)
Comment (NSDrowned) [reply packages/styles/scss/components/tabs-extended/_tabs-extended.scss:55]: @annawen1, the vertical text alignment was a bit off in the selected tab. It seems the line-height set by `productive-heading-01` is different than the other tabs. This is how it looks like without the overwrite:

image

It's 1px highter.

Comment (oliviaflory) [reply packages/styles/scss/components/tabs-extended/_tabs-extended.scss:55]: @NSDrowned thanks for fixing the text alignment! 

I think we could remove the line height override by switching the tab items to use $body-short-01. Carbon has some funky pairings with the heading and body styles that I didn't think about when I wrote the issue initially. 

Apologies for the back and forth, would we be able to remove the line height override in the heading style and switch from $body-long-01 to $body-short-01 ?

Comment (NSDrowned) [reply packages/styles/scss/components/tabs-extended/_tabs-extended.scss:55]: @oliviaflory I switched to `body-short-01` but still have the same problem. The line-height problem seems to lie in `productive-heading-01`.

Comment (NSDrowned) [reply packages/styles/scss/components/tabs-extended/_tabs-extended.scss:55]: Talked with Olivia over slack and we're going with the override for now 👍

**Comment by oliviaflory** [comment]
@NSDrowned overall looks really good!

I wasn't expecting the tabs Content type size to change between desktop and mobile (small & medium breakpoints)
Can we keep the small and medium at 16px (likely $body-long-02)

Screen Shot 2021-08-04 at 9 32 26 AM

**Review by oliviaflory — APPROVED** [review]
LGTM, thanks @NSDrowned ! 

Let's work together to open an issue for the Carbon type token line height error and make a note to come back and remove the overrides in Tabs extended later 👍

**Pull request merged**